app/org/sagebionetworks/bridge/services/ScheduledActivityService.java

```diff
@@ -164,26 +118,11 @@ public void deleteActivitiesForUser(String healthCode) {
     }
     
     protected List<ScheduledActivity> updateActivitiesAndCollectSaves(List<ScheduledActivity> scheduledActivities, List<ScheduledActivity> dbActivities) {
-        // BRIDGE-1589: After adjusting activity GUIDs' time portion, we will have generated db activities with duplicate 
-        // GUIDs in dbActivities, and Maps.uniqueIndex will throw an error because the GUIDs are not all unique. 
-        // Build the map manually while de-duplicating the db activities, particularly saving any copy that has 
-        // client-persisted state in it (startedOn is good enough here). 
-        // Map<String, ScheduledActivity> dbMap = Maps.uniqueIndex(dbActivities, ScheduledActivity::getGuid);
+        Map<String, ScheduledActivity> dbMap = Maps.uniqueIndex(dbActivities, ScheduledActivity::getGuid);
```

Comment:
Once this fix is rolled back, yes, there could be duplicated tasks that are visible, though it's been a month now and I'm wondering how many of these are left.